lualib/lua_content/pdf.lua

```diff
@@ -653,8 +653,8 @@ process_dict = function(task, pdf, obj, dict)
     if not obj.type then
 
       if obj.dict.S and obj.dict.JS then
-        obj.type = 'Javascript'
-        lua_util.debugm(N, task, 'implicit type for Javascript object %s:%s',
+        obj.type = 'JavaScript'
```

Comment:
If this change is a problem, I can drop it...

Reply:
Yes, there is a logic that checks if for `Javascript` so please revert.